### PR TITLE
github addons

### PR DESCRIPTION
**Pull request by davidegraff**
## Description
This PR adds a few github community integrations: issue templates, PR templates, a contribution guide

## Changes
None to the code, so no new unit tests have been added. Only new files have been added in the `.github/` directory

## Open questions
- there are PR templates added, but it seems that using them requires specifying the template in the URL of the PR. I don't like this approach, but it seems like it's the only way to do it

## Future work
- getting the UI working for PR templates
- more issue templates
- refactoring the github actions: IMO linting should be a separate job from testing. We can also add codecov, automatic `pip` packaging, auto-generating documentation, etc.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Making Changes
+
+* Add some really awesome code to your local fork.  It's usually a [good
+  idea](http://blog.jasonmeridth.com/posts/do-not-issue-pull-requests-from-your-master-branch/)
+  to make changes on a
+  [branch](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/)
+  with the branch name relating to the feature you are going to add.
+* When you are ready for others to examine and comment on your new feature,
+  navigate to your fork of pyscreener on GitHub and open a [pull
```
Comment (mliu49) [inline .github/CONTRIBUTING.md:22]: Replace pyscreener with Chemprop?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (mliu49) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Should these placeholder steps be adjusted to better fit chemprop?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment**
+- python version
+- package versions
```
Comment (mliu49) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: Perhaps explicitly request the output of conda list?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (davidegraff) [reply .github/CONTRIBUTING.md:22]: ope

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (davidegraff) [reply .github/ISSUE_TEMPLATE/bug_report.md]: how about the updated version?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -10,22 +10,18 @@ assignees: ''
...
 A clear and concise description of what the bug is.
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+**Example(s)**
+Provide some examples of where the current code fails. Feel free to share your actual code for additional context, but a minimal and isolated examlpe is preferred.
```
Comment (mliu49) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: `examlpe -> example`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+## Bugfix / Desired workflow
+Include either **(a)** the same workflow from above with the correct output produced via this PR **(b)** some (pseudo)code containing the new workflow that this PR will (seek to) implement
+
+## Questions
+If there are open questions about implementation strategy or scope of the PR, include them here
+
+## Relevant issues
+If appropriate, please tag them here and include a quick summary
+
+## checklist
```
Comment (mliu49) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: Capitalize Checklist for consistency with other headings? (also for other two pull request templates)

**Comment by mliu49** [comment]
Thanks for making the changes! I noticed two other minor things, but otherwise it looks good to me.

**Comment by davidegraff** [comment]
thanks @mliu49! just pushed the new changes. Any other thoughts on the future work section re: including in this PR or opening relevant issues?

**Comment by mliu49** [comment]
For the PR template UI, there doesn't seem to be much we can do, until GitHub adds support? Perhaps put links with the query parameters in the README or something, but that doesn't seem great.

For more issue templates, did you have any in mind? I think one possibility could be a general question template (e.g. https://github.com/ReactionMechanismGenerator/RMG-Py/blob/main/.github/ISSUE_TEMPLATE/question.md), unless we want to make use of the Discussions tab for those instead.

For the improvements to CI pipeline, I agree with all of the suggestions. Those changes should be a separate PR though, and it might be good to create some issue(s) first.

**Comment by davidegraff** [comment]
a general question template is a good idea- I'll add that in. Pending review of that addition, this PR should be good to merge?

**Comment by cjmcgill** [comment]
It looks good by me, but @mliu49 has engaged with it more so I'll wait for his review.

**Review by cjmcgill — APPROVED** [review]
Looks good. I will merge.

**Pull request merged**